sdk/python/feast/feature_store.py

```diff
@@ -1172,6 +1176,15 @@ def get_online_features(
                 except KeyError as e:
                     raise ValueError("All entity_rows must have the same keys.") from e
 
+        # If Go feature server is enabled, send request to it instead of going through a regular Python logic
+        if enable_go_feature_server(self.config):
+            # Lazily start the go server on the first request
+            if self._go_server is None:
+                self._go_server = GoServer(str(self.repo_path.absolute()), self.config)
```

Comment:
Perhaps we can delay casting to `str` until the last moment?

Reply:
Can you clarify on what the last moment is?

Reply:
https://github.com/feast-dev/feast/pull/2339/files#diff-9ef7735ea7a5e6ea2b8ad263b7e3dd5c51ba67e29de9f3b2f34eacc063e0cebbR146